index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
@LobeTia it would be very dirty solution.
Because `onSchema` key in `schema()` method will confuse a lot. It's much easier to add new method and all that is related to `mongoose` would be in this method. I think it much better rather than new special key in `schema()`